packages/logger/src/index.ts

```diff
@@ -56,13 +55,7 @@ class Logger {
 
         return item;
       })
-      .map(item => {
-        if (typeof item === 'object') {
-          return JSON.stringify(item);
-        }
-
-        return item;
-      })
+      .map(item => JSON.stringify({ tags: this.tags, log: item }))
```

Comment:
set log field to `message`, so log by default would look like
```
{ "tags": [ ... ], "message": ... }
```
and we will search it in elastic by `log.message.*`

Reply:
changes has been applied

```
error:test {"tags":["graphql"],"message":"Hello world!"}
error:test {"tags":["graphql"],"message":"Hello world!","code":"403"}
```